### ISSUE TITLE
coveralls.io overstating test coverage?

### ISSUE THREAD

**Issue by bgourlie** [shaneutt/riak-rust-client#1]
I ran across the blog post documenting how to configure rust projects with coveralls. In looking into the solution used by Riak, I noticed that it's showing 100% test coverage for all files. Looking at the coverage of individual files seems to defy this, for example: https://coveralls.io/builds/9659146/source?filename=bucket.rs

There multiple functions that aren't covered, so I'm a bit confused why it's showing 100% coverage. Do I just have a fundamental misunderstanding of how coverage stats are determined?

**Comment by shaneutt** [comment]
You are correct that the test coverage is not represented properly, and this is something that needs to be further investigated and corrected.
